internal/modules/staking/test.py

```diff
@@ -16,12 +16,16 @@
 
 HOME = os.getenv("HOME")
 DAEMON = os.getenv("DAEMON")
+DAEMON_HOME = os.getenv("DAEMON_HOME")
+os.environ["node1_home"] = f"{DAEMON_HOME}-1"
+os.environ["node2_home"] = f"{DAEMON_HOME}-2"
```

Comment:
This should be done inside the env setup before starting the n/w, so that we can use it wherever needed